.github/scripts/build

```diff
@@ -23,4 +23,8 @@ if [[ -z "$SBT_COMMAND" ]]; then
     exit 1
 fi
 
-#exec sbt ++$SCALA_VERSION $SBT_COMMAND
+if [ $SCALA_VERSION == '2.11.12' ] && [ $SCALAJS_VERSION == '1.0.0' ]; then
+echo "No tests executed since sjs 1.0.0 and scala 2.11 are not compatible."
```

Comment:
Notice the workaround here that checks versions before running tests in case they are incompatible, tried to add that check just before to trigger the job but found out that it was not possible to access matrix values [before the initialisation](https://github.community/t5/GitHub-Actions/Matrix-cannot-be-used-in-jobs-level-if/td-p/41048).

Reply:
> Nice! How many tests can run concurrently? If not all, we could remove some configurations, e.g. don't test each Scala version with both jdk versions

So I guess all of them can run concurrently, right now there is a dependency in which scalajs tests does not start until jvm have not finished, but will remove this logic since it does not makes much sense.

Reply:
Removed that dependency, definitely they all run in parallel :D

Reply:
Note that this PR is not pointing to `master` but to `github-actions`. So before it goes to master, the secrets for automated releases needs to be [configured](https://github.com/olafurpg/sbt-ci-release).